index.html

```diff
@@ -0,0 +1,192 @@
+		crossorigin="anonymous"
+	/>
+			<link rel="stylesheet" href="styles.css">
+			<script src="https://kit.fontawesome.com/e98a0f000a.js" crossorigin="anonymous"></script>
+		
+			<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+			<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Zenith Bootcamp</title>
+</head>
+<body>
+	<nav class="menu-wrap">
+		<input type="checkbox" class="toggler">
+		<div class="hamburger"><div></div></div>
+		<div class="menu">
+			<div class="one">
+				<div class="two">
+					<div class="nav1">
+						<ul class="nav1-items">
+						  <li class="d-flex">
+								<i class="nav-link nav1-link fab fa-facebook-f"></i>
+								<i class="nav-link nav1-link fab fa-twitter"></i>
+						  </li>
+						  <li><select name="languages" class="languages" id="langs">
+							  <option class="language" value="english">English</option>
+							  <option class="language" value="spanish">Spanish</option>
+							  <option class="language" value="french">French</option></select>
+					    </li>
+						  <li><a class="nav1-item nav-link" href="#">My page</a></li>
+						  <li><a class="nav1-item nav-link nav1-logout" href="#">Logout</a></li>
+						</ul>
+					</div>
+					<div class="nav2">
+						<div class="logo">
+							<img src="img/zenith.jpeg" alt="zenith logo">
+							<h6 class="logo-text">Bootcamp</h6>
+						</div>
+						<ul class="nav2-items">
+							<li><a class="about nav-link" href="about.html">About</a></li>
+							<li><a class="nav2-link nav-link" href="#">Program</a></li>
+							<li><a class="nav2-link nav-link" href="#">Join</a></li>
+							<li><a class="nav2-link nav-link" href="#">Sponsor</a></li>
+							<li><a class="nav2-link nav-link" href="#">News</a></li>
+							<li><a class="campaign nav-link" href="#">Campaign</a></li>
+						</ul>
+					</div>
+				</div>
+			</div>
+		</div>
+	</nav>
+	 <header class="showcase">
+			<h1 class="desc">Bootcamp school to become a full Stack web developer in 2021</h1>
+			 <p class="para">Browse through the website for the list of the greatest courses created in programming. Join our bootcamp and start learning most of the highest quality courses ever created in programming.</p>
+			 <h2>2021.10.15(THU) ~ 16(FRI)</h2>
+			 <p class="para1">Courses on Web Development, Back End, Front End, and More</p>
+	 </header>
+	 <section class="program">
+		 <h2 class="program-text">Main program</h2><hr>
+		 <div class="courses">
+			 <div class="html pro">
+				 <img class="img1 img-logo" src="img/html-folder1.jpeg" alt="html">
+				 <h5>HTML</h5>
+				 <p> Lorem ipsum dolor sit amet consectetur adipisicing elit. Molestiae dolores quis voluptate atque similique ullam. Sunt, quidem voluptatem quod eaque. </p>
```

Comment:
- [ ] As stated before, kindly add some margin/padding between the `image-logo `and the text beneath it. There is also some padding lacking above the logo. Seems you have also changed the color of the `<h5>` element which should remain as of the given design.
the given design:
![image](https://user-images.githubusercontent.com/68167810/133749550-120d2d4e-7277-4228-9b01-c1af8d65fb9d.png)
your design:
![image](https://user-images.githubusercontent.com/68167810/133749379-b0a8f335-175a-45b3-a590-8d2fbfae379d.png)
- [ ] The text in the `<p>` tag should be centered as also stated earlier. Yours is aligned to the left kindly fix this.
the given design:
![image](https://user-images.githubusercontent.com/68167810/133748489-89ae6aa9-c93b-45cb-889e-62d3f54c9cb6.png)
=================================================

your design: 
![image](https://user-images.githubusercontent.com/68167810/133748645-42368e69-1125-4738-8b8a-71443086025d.png)
- [ ] Remember to also reduce the padding below at the bottom of each card.